src/lib/cli/rules/templates/external-rule/files/.editorconfig

```diff
@@ -0,0 +1,23 @@
+# For more information about the configurations used
```

Comment:
This file is not needed since it doesn't provide any user value, and all the configurations will be inherited from the root `.editorconfig`

Reply:
In an external rule, there is no root `.editorconfig`. But I can remove it if you want to :)

Reply:
Nevermind, ignore my previous comments, I misunderstood the scope of this PR.